include/arch/arm/armv/armv8-a/64/armv/benchmark.h

```diff
@@ -19,16 +19,12 @@
 static inline void armv_enableOverflowIRQ(void)
 {
     uint32_t val;
-    MRS(PMINTENSET, val);
+    MRS("PMINTENSET_EL1", val);
```

Comment:
Will this work if the kernel is running hypervisor mode? And hence EL2

Reply:
EL2 should have access to EL1 registers I believe.